update-ov.sh

```diff
@@ -35,16 +35,23 @@ if [ "$2" == "tomcat" ]; then
     ARTIFACT_CLASSIFIER=""
 
     SERVER_XML_FILE="$TOMCAT_PATH/conf/server.xml"
+    CONTEXT_PATH="$TOMCAT_PATH/conf/Catalina/$WEBSITE"
+    CONTEXT_XML_FILE="$CONTEXT_PATH/ROOT.xml"
 
     HOST_XPATH="Service/Engine[@name=\"Catalina\"]/Host[@name=\"$WEBSITE\"]"
-
-    APP_BASE="$(read_xml_value "$SERVER_XML_FILE" "$HOST_XPATH" "appBase")"
-    if [ -z "$APP_BASE" ]; then
+    if [ -z "$(read_xml_value "$SERVER_XML_FILE" "$HOST_XPATH" "name")" ]; then
         echo "No website with name [$WEBSITE] is available!"
         exit 1
     fi
 
-    WEBAPP_PATH="$TOMCAT_PATH/$APP_BASE"
+    DOC_BASE="$(read_xml_value "$CONTEXT_XML_FILE" "" "docBase")"
+    if [ -z "$DOC_BASE" ]; then
+        echo "No 'docBase' attribute for website with name [$WEBSITE] is defined!"
+        exit 1
+    fi
+
+    WEBAPP_PATH="${DOC_BASE/\$\{catalina.base\}/$TOMCAT_PATH}"
+    WEBAPP_PATH="${DOC_BASE/\$\{catalina.home\}/$TOMCAT_PATH}"
```

Comment:
why do you assign it twice?

Reply:
1. Replace catalina.base
2. Replace catalina.home

Reply:
I found that first replacement overrides by second assignment, this is fixed.